src/OSPSuite.Core/Domain/Builder/SimulationConfiguration.cs

```diff
@@ -35,9 +35,17 @@ public class SimulationConfiguration : IVisitable<IVisitor>, IUpdatable
 
       public IReadOnlyList<T> All<T>() where T : class, IBuildingBlock
       {
+         if (typeof(T) == typeof(IBuildingBlock))
```

Comment:
asking for All<IBuildingBlock> would not return all of them.. this was a subtle bug